mesh/CMakeLists.txt

```diff
@@ -48,17 +51,23 @@ set(HDRS
   vertex.hpp
   vtk.hpp
   wedge.hpp
+  submesh/transfer_category.hpp
+  submesh/transfermap.hpp
```

Comment:
It looks like `submesh.hpp` and `submesh_utils.hpp` are not added to the list of headers. I just wanted to check if this was an oversight or intentional.